src/coreclr/src/System.Private.CoreLib/ILLink.targets

```diff
@@ -1,146 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There is important logic in this file that auto-generates the ILLinkXml file based on the unmanaged runtime dependencies. It is not ok to just delete it.

If you want to refactor this or clean it up, it would be best done as separate PR.

Reply:
I diffed the contents and went through git history. It's a duplicate of eng/illink.targets.

Reply:
The special thing that you are referring to is probably https://github.com/dotnet/runtime/blob/master/src/coreclr/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets which will stay as is.